docs/src/main/asciidoc/security-webauthn.adoc

```diff
@@ -87,7 +87,7 @@ The solution is located in the `security-webauthn-quickstart` {quickstarts-tree-
 First, we need a new project. Create a new project with the following command:
 
 :create-app-artifact-id: security-webauthn-quickstart
-:create-app-extensions: security-webauthn,reactive-pg-client,resteasy-reactive,hibernate-reactive-panache,test-security-webauthn
+:create-app-extensions: security-webauthn,reactive-pg-client,resteasy-reactive,hibernate-reactive-panache
```

Comment:
Have you tested if this actually adds the library or if it ignores it?

Reply:
Is throwing an error!
```
quarkus create app org.acme:security-webauthn-quickstart \
    --extension='security-webauthn,reactive-pg-client,resteasy-reactive,hibernate-reactive-panache,test-security-webauthn' \
    --no-code
Looking for the newly published extensions in registry.quarkus.io
[ERROR] ❗  Cannot find a dependency matching 'test-security-webauthn', maybe a typo?
[ERROR] ❗  Unable to create project: Failed to create project because of invalid extensions

```

Reply:
Same if you try to add the library later on:
```
quarkus extension add 'test-security-webauthn'
[ERROR] ❗  Nothing installed because keyword(s) 'test-security-webauthn' were not matched in the catalog.

```

Reply:
but... I like your idea of ignore those dependencies that are not in the catalog!. Maybe if an extension is not in the catalog we could just  write a warning message but create the app with those deps that are right.

Reply:
Could you try adding `quarkus-test-security-webauthn` instead?

I think it's better to keep it an error so if not working at all, I will merge your patch.